src/main/java/com/example/techiteasyspringhomework/controllers/TelevisionController.java

```diff
@@ -11,7 +11,7 @@
 
 @RestController
 @RequestMapping("/televisions")
-public class TelevisionsController {
+public class TelevisionController {
 
     private ArrayList<Television> televisions = new ArrayList<>();
```

Comment:
We willen hier geen arraylist meer maar de repository :) (stond niet in de opdracht volgens mij)